config/config.go

```diff
@@ -68,11 +68,26 @@ type Build struct {
 	DiskSize    int
 	OEMSize     string
 	OEMFSSize4K uint64
+	SealOEM     bool
 	GPUType     string
 	Timeout     string
 	GCSFiles    []string
 }
 
+// Update clears the config file and then saves the new config.Build.
+func Update(configFile *os.File, buildConfig *Build) error {
+	if _, err := configFile.Seek(0, 0); err != nil {
+		return fmt.Errorf("cannot seek config file, error msg:(%v)", err)
```

Comment:
nit: Since this function is specific to the build config, it might be good to say "build config" instead of "config" in the error message.